mvp-spec/json-schemas/seed.json

```diff
@@ -19,13 +28,15 @@
             ]
         },
         "source": {
-            "description": "Signature based on input:\n```seed.source.domain + '\\u2063' +\nseed.source.timestamp + '\\u2063' +\nseed.transaction_id + '\\u2063' +\nseed.publisher + '\\u2063' +\ndata.identifiers[0].source.signature + '\\u2063' +\ndata.identifiers[1].source.signature + '\\u2063' +\n... + '\\u2063' +\ndata.identifiers[n].source.signature + '\\u2063' +\ndata.preferences.source.signature\n```",
```

Comment:
You removed the `seed.source.domain`, I guess this is a mistake.

Reply:
On purpose. I simplify it.

Reply:
I just remove the `seed` prefix to simplify the description,

Reply:
Oh yes my bad, sorry.